src/plone/restapi/types/utils.py

```diff
@@ -139,7 +140,10 @@ def get_tagged_values(schemas, key):
         for field_name in schema:
             widget = tagged_values.get(field_name)
             if widget and widget.params:
-                params[field_name] = widget.params
+                params[field_name] = widget.params.copy()
```

Comment:
@davisagli somehow "tagged_values.get(field_name)" returns a string in Plone 4.3/5.0 which makes the test on Travis.

Reply:
@davisagli what's the state of this PR?

There are two issues with this PR:

1. different versions for `plone.app.event`

your assertion will fail with `plone.app.event` < `3.0` because there is no default timezone defined for the fields (https://github.com/plone/plone.app.event/blob/3.0/plone/app/event/dx/behaviors.py#L112). `plone.app.event`< '3.0` will be used by `Plone` <= `5.0`

2. widget-string

The issue mentioned by @tisto is another issue.

See https://github.com/plone/plone.restapi/issues/664

Warning: `Plone` > `5` uses `plone.app.event`> `3.0` which is using a `ParametrizedWidget` for its fields (https://github.com/plone/plone.app.event/blob/3.x/plone/app/event/dx/behaviors.py#L86). So this issue wont affect Plone > 5.

Reply:
... I pushed to this PR with a fix.